docs/en/observability/monitor-azure-native.asciidoc

```diff
@@ -0,0 +1,148 @@
+[[monitor-azure-native]]
+== Monitor Microsoft Azure with the native Azure integration
+
+****
+The Elastic Cloud native Azure integration allows you to deploy managed
+instances of the {stack} directly in Azure, through the Azure integrated
+marketplace. The integration includes native capabilities for consolidating
+Azure logs and metrics in Elastic. For more information, refer to
+https://www.elastic.co/guide/en/cloud/current/ec-azure-marketplace-native.html[Native Azure integration]
+in the {ess} documentation.
+
+**Using {agent} to monitor Azure?** Refer to <<monitor-azure-elastic-agent>>.
+
+****
+
+In this tutorial, you'll learn how to:
+
+* Create an {es} resource in the Azure portal.
+* Ingest Azure platform logs using the native integration.
+* Ingest logs and metrics from your virtual machines.
+* Visualize the logs and metrics in {kib}.
+
+[discrete]
+[[azure-create-resource]]
+=== Step 1: Create an {es} resource in the Azure portal
+
+IMPORTANT: These steps will not work if you have an active GCP or AWS deployment
+in {ecloud} that is already associated with the email address used for your
+Azure account. To avoid this problem, delete your GCP and AWS deployments in
+{ecloud}, or use a different Azure account. If this does not resolve your issue,
+reach out to us at support@elastic.co.
```

Comment:
If we suggest a time, we should be specific like...wait nn hours. Is 24 hours enough? 12 hours?

Reply:
I'm leaning towards 12 hours, but @csierra may have a better take on this.

Reply:
hi folks... I am not sure I am the most knowledgeable person about this... I don't really know how long could it take since a deployment is requested to shutdown until it actually does shutdown completely for the rest of the systems

Reply:
@csierra Is there some way for users to make sure the deployment is shutdown completely? Otherwise I think we'll need to be really vague in the docs (not a great user experience).

Reply:
@dedemorton I think I found a bug in the logic accessing the existing deployments that may explain the delay that you have experienced. 
Let me make a couple of inquiries and I will come back to you folks on this.

Reply:
so I am not sure if it's a bug but I believe the problem is that we seem to consider terminated deployments that are NOT hidden as incompatible. So even if you stopped your deployment, if it was not hidden, it was probably still considered as incompatible and yielding an error.

Reply:
I have tried this situation myself and it seems it works instantaneously if the user deletes their deployment from the cloud console. Once the deployment reads that it is hidden it is no longer taken into account by the marketplace validation and it just passes through. 
I have successfully tried this in staging.

Reply:
Thanks for following up on this! In my case, I simply deleted the deployments, but it sounds like there might have been an unexpected glitch. I will leave the tutorial as-is for now and update it later if we get feedback indicating that users are running into the same problem I encountered.